### PR TITLE
Datepicker input will now show default placeholder of date format

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-datepicker-placeholder](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-datepicker-placeholder)

File: src/app/public/modules/datepicker/datepicker-adapter.service.ts [hunk 1]
```diff
@@ -26,4 +26,12 @@ export class SkyDatepickerAdapterService {
 
     return styles && styles.visibility === 'visible';
   }
+
+  public getPlaceholder(elementRef: ElementRef): string {
+    return elementRef.nativeElement.getAttribute('placeholder');
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/datepicker/datepicker-adapter.service.ts:36]: Good thought. Done.

File: src/app/public/modules/datepicker/datepicker-input.directive.ts [hunk 2]
```diff
@@ -255,6 +255,7 @@ export class SkyDatepickerInputDirective
   private control: AbstractControl;
   private dateFormatter = new SkyDateFormatter();
   private isFirstChange = true;
+  private initalPlaceholder: string;
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/datepicker/datepicker-input.directive.ts]: Fixed.

**Comment by Blackbaud-AlexKingman** [comment]
@Blackbaud-SteveBrush ready for you again.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-datepicker-placeholder](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-datepicker-placeholder)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**